[2021-08-12 12:09:53,132] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: pipeline_try.thursday_Task 2021-08-12T12:09:39.723577+00:00 [queued]>
[2021-08-12 12:09:53,227] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: pipeline_try.thursday_Task 2021-08-12T12:09:39.723577+00:00 [queued]>
[2021-08-12 12:09:53,230] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-12 12:09:53,232] {taskinstance.py:1088} INFO - Starting attempt 1 of 2
[2021-08-12 12:09:53,237] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-12 12:09:53,272] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): thursday_Task> on 2021-08-12T12:09:39.723577+00:00
[2021-08-12 12:09:53,292] {standard_task_runner.py:52} INFO - Started process 7306 to run task
[2021-08-12 12:09:53,310] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'pipeline_try', 'thursday_Task', '2021-08-12T12:09:39.723577+00:00', '--job-id', '65', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_try1.py', '--cfg-path', '/tmp/tmpr1jwrxks', '--error-file', '/tmp/tmp6z0jbo01']
[2021-08-12 12:09:53,324] {standard_task_runner.py:77} INFO - Job 65: Subtask thursday_Task
[2021-08-12 12:09:53,462] {logging_mixin.py:104} INFO - Running <TaskInstance: pipeline_try.thursday_Task 2021-08-12T12:09:39.723577+00:00 [running]> on host 99300daf4d26
[2021-08-12 12:09:53,612] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=ameysonje242000@gmail.com
AIRFLOW_CTX_DAG_OWNER=amey
AIRFLOW_CTX_DAG_ID=pipeline_try
AIRFLOW_CTX_TASK_ID=thursday_Task
AIRFLOW_CTX_EXECUTION_DATE=2021-08-12T12:09:39.723577+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-08-12T12:09:39.723577+00:00
[2021-08-12 12:09:53,621] {logging_mixin.py:104} INFO - From thursday_Task, so its thursday today
[2021-08-12 12:09:53,622] {python.py:151} INFO - Done. Returned value was: email
[2021-08-12 12:09:53,694] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=pipeline_try, task_id=thursday_Task, execution_date=20210812T120939, start_date=20210812T120953, end_date=20210812T120953
[2021-08-12 12:09:53,786] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-08-12 12:09:53,847] {local_task_job.py:149} INFO - Task exited with return code 0
